lib/engine/depot.rb

```diff
@@ -12,7 +12,7 @@ def initialize(trains, game)
       @trains.each { |train| train.owner = self }
       @upcoming = @trains.dup
       @discarded = []
-      @bank = @game.bank
+      @bank = game.cobank
```

Comment:
this is wrong, it should be the game.bank or a random bank you pass in right?

Reply:
Ok, so the idea I'm not explaining very well is that corporations would (in general) interact with one bank and that players would interact with another. In many games (all except 1860 at this point), the two would be set to the same bank in Game::Base, but in 1860 they would be separate. So I need two different bank names depending on whether it's a corporation doing the spending/receiving or not.